docker-compose.yml

```diff
@@ -5,10 +5,6 @@ services:
   node_alice:
     container_name: alice
     image: "parity/polkadot:v0.9.18"
-    ports:
```

Comment:
We still need this port mapping right? It is the mapping of container to local.

Reply:
Or does using the `host` networking change this?

Reply:
Yes running on host, so port wont be needed in compose